frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -134,6 +139,77 @@ function AddClusterButton() {
   );
 }
 
+function SidebarToggleButton() {
```

Comment:
Probably should be in a SidebarToggleButton.tsx with corresponding stories.

Reply:
I think we have a story for the full sidebar so this is covered. I prefer small modules to be contained in the more main ones.